config/rethinkdb_logstash.conf

```diff
@@ -11,12 +11,17 @@ input {
 
 filter {
   mutate {
-    add_field => {"type" => "volunteer"}
+    add_field => {"type" => "volunteer"} 
     rename => {
       "new_val" => "[doc]"
     }
+    add_field => {"suggest" => "%{[doc][suggest]}"} 
```

Comment:
Ogólnie potrzebujemy pola suggest dla elastic search, aby działały sugestie wolontariuszy.
Pole `suggest` jest formatu 
```
suggest: {
  input: [first_name, last_name], 
  output: first_name+" "+last_name,
  payload: {id: "jakies id", email: "jakis email"}
}
```
Wydawało mi się, że łatwiej jest utworzyć taką strukturę w javascript, dodać ją do obiektu wolontariusza a następnie skopiować to poprzez "suggest" => "%[doc][suggest]". Ale mogę się zastanowić, czy da się taką strukturę `suggest` utworzyć bezpośrednio w rethinkdb_logstash.conf.